package.json

```diff
@@ -48,17 +48,17 @@
     "require-dir": "^1.2.0"
   },
   "peerDependencies": {
-    "eslint": "^7.0.0",
-    "typescript": "^3.7.0 || ^4.0.0"
+    "eslint": "^8.36.0",
```

Comment:
i didnt realize we had merged the eslint@8 pr. need to run nightman to check if this works

Reply:
It's not from Master. I only pulled the beachball config files from master

Reply:
The update was done by Bill. While resolving merge conflicts it somehow got messed up. So I updated to eslint8 again and generated the lock file

Reply:
Weird, I'm pretty sure it was 8.36.0 before I merged master. @wgoehrig could you verify please?

Reply:
@yashincontrol  you can verify by looking through the history 
https://github.com/iTwin/eslint-plugin/commits/ts5/package.json